src/test/resources/project/Java.template

```diff
@@ -0,0 +1,79 @@
+package <package_name>;
```

Comment:
To be simplified. Also, why do we have two template directories ? one in `main` and one in `test` ?